site/content/blogs/apprunner-waf.en.md

```diff
@@ -0,0 +1,140 @@
+# Using WAF With App Runner in Copilot
+
+Posted On: Feb 08, 2023
+
+**Siddharth Vohra**, Software Development Engineer, AWS App Runner
+
+You can now associate your AWS Web Application Firewall (WAF) Web access control lists (web ACLs) with your App Runner service - all in Copilot!
+
+[AWS Web Application Firewall (WAF)](https://docs.aws.amazon.com/waf/latest/developerguide/waf-chapter.html) helps you monitor the HTTP(S) requests that are forwarded to your web applications,
+and allows you to control access to your content. 
+You can protect your application by blocking or allowing web requests based on criteria that you specify, 
+such as the IP addresses that requests originate from or the values of query strings.
+
+Today, AWS announced the WAF support for AWS App Runner. 
```

Comment:
An idea: if there is a blog post that AppRunner + WAF is making, we can add the link to that blog post here as well!
```suggestion
Today, AWS announced the [WAF support for AWS App Runner](/link/to/post). 
```

Reply:
Looks like we'll only be able to do this with a new pull request early next week.